File: Stack And Queues/Remove Outermost Parentheses/RemoveOutermostParentheses.cpp
#include <iostream>
#include <stack>
using namespace std;

string removeOuterParentheses(string s) {
    stack stack;
    string ans = "";

    for (int i = 0; i < s.length(); i++) {
        if (s[i] == '(') {
            stack.push(s[i]);
            if (stack.size() > 1) ans += s[i];
        } else {
            if (stack.size() > 1) ans += s[i];
            stack.pop();
        }
    }
    return ans;
}

File: Stack And Queues/Remove Outermost Parentheses/RemoveOutermostParentheses.js
var removeOuterParentheses = function(s) {
    let stack = [];
    let ans = "";

    for(let i=0; i 1) ? s[i] : "");
        } else {
            ans += ((stack.length > 1) ? s[i] : "");
            stack.pop();
        }
    }
    return ans;
};